### PR TITLE
The whole app

### PR DESCRIPTION
**Pull request by Olamarx**
- Created the react app with 'npx create-react-app'
- Added the readme file
- Added the linter checkers
- Added component files
- Added some functions such as the onChange.
- Styled the UI

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,31 +1,46 @@
...
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+ - React
+
+## Live Demo
+
+[Live Demo Link(Yet to be pulished)]()
```
Comment (Addisu87) [inline README.md:16]: Descriptive readme file

File: src/App.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+          <code>src/App.js</code>
+          {' '}
+          and save to reload.
+        </p>
+        <a
+          className="App-link"
+          href="https://reactjs.org"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
```
Comment (Addisu87) [inline src/App.js]: Please remove unnecessary inputs.

File: src/App.test.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
+  render(<App />);
+  const linkElement = screen.getByText(/learn react/i);
+  expect(linkElement).toBeInTheDocument();
+});
```
Comment (Addisu87) [inline src/App.test.js]: Please remove unused files.

File: src/components/TodoItem.js [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+          type="checkbox"
+          className={style.checkbox}
+          checked={this.props.todo.completed}
+          onChange={() => this.props.handleChangeProps(this.props.todo.id)}
+        />
+        <span style={this.props.todo.completed ? completedStyle : null}>
+          {this.props.todo.title}
+        </span>
+        <button
+          onClick={() => this.props.deleteTodoProps(this.props.todo.id)}
```
Comment (Addisu87) [inline src/components/TodoItem.js:29]: Use destructuring methods for this.props.todo.

File: src/reportWebVitals.js [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+      getCLS(onPerfEntry);
+      getFID(onPerfEntry);
+      getFCP(onPerfEntry);
+      getLCP(onPerfEntry);
+      getTTFB(onPerfEntry);
+    });
+  }
+};
+
+export default reportWebVitals;
```
Comment (Addisu87) [inline src/reportWebVitals.js]: - Please remove this file.
- Please add some more functionality to your file and try to use the function component method.

**Review by Addisu87 — COMMENTED** [review]
It is better to practice using hooks and function component method. Overall, your code is clean and DRY. 👏💯.

File: README.md (hunk 1 above)
Comment (Olamarx) [reply README.md:16]: Hello @Addisu87,
please can you be explicit on what you mean by descriptive readme file?

File: src/App.js (hunk 2 above)
Comment (Olamarx) [reply src/App.js]: @Addisu87 thank you, I have removed the file.

File: src/App.test.js (hunk 3 above)
Comment (Olamarx) [reply src/App.test.js]: @Addisu87 thank you, I have removed it.

File: src/reportWebVitals.js (hunk 5 above)
Comment (Olamarx) [reply src/reportWebVitals.js]: @Addisu87 Thank you for the review, I have removed the file. I will also add more functions.

File: src/components/TodoItem.js (hunk 4 above)
Comment (Olamarx) [reply src/components/TodoItem.js:29]: @Addisu87 Thank you.

**Pull request merged**